App/Sagas/ImageSharingSagas.ts

```diff
@@ -20,8 +20,8 @@ import TextileNodeActions, { TextileNodeSelectors } from '../Redux/TextileNodeRe
 import { ActionType, getType } from 'typesafe-actions'
 import NavigationService from '../Services/NavigationService'
 import * as CameraRoll from '../Services/CameraRoll'
-import { IMobilePreparedFiles } from '../NativeModules/Textile/pb/textile-go'
-import { waitFor } from './WaitFor'
+import { IMobilePreparedFiles } from '@textile/react-native-protobufs'
```

Comment:
`react-native-sdk` isn't usable without `react-native-protobufs` and `react-native-protobufs` doesn't do anything without `react-native-sdk`, so they could just be combined into a single module.